containers/server-image/Dockerfile

```diff
@@ -73,16 +78,23 @@ COPY java_agent.yaml /etc/prometheus-jmx_exporter/tomcat/java_agent.yml
 COPY java_agent.yaml /etc/prometheus-jmx_exporter/taskomatic/java_agent.yml
 COPY tomcat_jmx.conf /usr/lib/systemd/system/tomcat.service.d/jmx.conf
 COPY taskomatic_jmx.conf /usr/lib/systemd/system/taskomatic.service.d/jmx.conf
+COPY pam_susemanager /etc/pam.d/susemanager
```

Comment:
file needs to be readable by tomcat cat. Which access will this file have?

Reply:
```
-rw-r--r-- 1 root root 164 Apr  9 09:33 /etc/pam.d/susemanager
```
do you think it's fine or is it better to have other permission?

Reply:
all good. tomcat user only needs to be able to read it, and with that permissions is fine.